jsonb-generator/src/main/java/io/avaje/jsonb/generator/SimpleAdapterWriter.java

```diff
@@ -108,6 +108,9 @@ private void writeClassEnd() {
   }
 
   private void writeClassStart() {
+    if (genericParamsCount > 0) {
+      writer.append("@SuppressWarnings({\"unchecked\",\"rawtypes\"})").eol();
```

Comment:
ahh, but can we add a space between `\"unchecked\",\"rawtypes\"}` to make it `\"unchecked\", \"rawtypes\"}`